src/crimson/osd/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ add_executable(crimson-osd
   watch.cc
   )
 target_link_libraries(crimson-osd
-  crimson-common crimson-os crimson fmt::fmt)
+  ceph-common crimson-common crimson-os crimson fmt::fmt)
```

Comment:
crimson is a project which tries to avoid linking against ceph-common. please don't do this.

Reply:
I wish I could, but then we'll have unresolved symbols.

Reply:
crimson is based on the seastar library which does not support Windows. so if we go with the win32::dl library approach we can just avoid linking against it.

Reply:
Yes, you're right.